assets/lang/pt_BR.ini

```diff
@@ -1,1218 +1,1219 @@
+translators2 = Felipe
+translators3 = 
+translators4 = 
+translators5 = 
+translators6 = 
+Twitter @PPSSPP_emu = Twitter @PPSSPP_emu
+website = Verifique o site da web:
+written = Escrito em C++ pela velocidade e portabilidade
+
+[MemStick]
+Already contains PSP data = Já contém dados do PSP
+Create or Choose a PSP folder = Criar ou Escolher uma pasta do PSP
+Current = Atual
+DataCanBeShared = Os dados podem ser compartilhados entre o PPSSPP regular/Gold
+DataCannotBeShared = Os dados NÃO PODEM ser compartilhados entre o PPSSPP regular/Gold!
+DataWillBeLostOnUninstall = Aviso! Os dados serão perdidos quando você desinstalar o PPSSPP!
+DataWillStay = Os dados permanecerão mesmo se você desinstalar o PPSSPP.
+Done! = Concluído!
+EasyUSBAccess = Acesso fácil ao USB
+Failed to move some files! = Falhou em mover alguns arquivos!
+Failed to save config = Falhou em salvar a configuração
+Free space = Espaço livre
+Manually specify PSP folder = Especificar manualmente a pasta do PSP
+MemoryStickDescription = Escolher aonde manter os dados do PSP (Cartão de Memória)
+Move Data = Mover Dados
+Selected PSP Data Folder = Pasta Selecionada dos Dados do PSP
+Skip for now = Ignorar por enquanto
+No data will be changed = Nenhum dado será mudado
+PPSSPP will restart after the change = O PPSSPP reiniciará após a mudança
+Starting move... = Iniciando a movimentação...
+That folder doesn't work as a memstick folder. = Essa pasta não funciona como um cartão de memória.
+USBAccessThrough = Acesso ao USB através do Android/data/org.ppsspp.ppsspp/files
+USBAccessThroughGold = Acesso ao USB através do Android/data/org.ppsspp.ppssppgold/files
+Use App Private Data = Use os Dados Privados do App
+Use PSP folder at root of storage = Use a pasta do PSP na raiz da armazenagem
+Welcome to PPSSPP! = Bem-vindo ao PPSSPP!
+WhatsThis = O que é isto?
+
+[RemoteISO]
+Browse Games = Procurar jogos
+Local Server Port = Porta do servidor local
+Manual Mode Client = Configurar o cliente manualmente
+Remote disc streaming = Streaming remoto do disco
+Remote Port = Porta remota
+Remote Server = Servidor remoto
+Remote Subdirectory = Sub-diretório remoto
+RemoteISODesc = Os jogos na sua lista de "Recentes" serão compartilhados
+RemoteISOLoading = Conectado, carregando a lista de jogos...
+RemoteISOScanning = Escaneando... Cliqme em "compartilhar jogos" no seu dispositivo do servidor
+RemoteISOScanningTimeout = Escaneando... Verifique suas configurações do firewall na área de trabalho
+RemoteISOWifi = Nota: Conecte os dois dispositivos na mesma rede Wi-Fi
+RemoteISOWinFirewall = AVISO: O Firewall do Windows está bloqueando o compartilhamento
+Settings = Configurações
+Share Games (Server) = Compartilhar jogos (servidor)
+Share on PPSSPP startup = Compartilhar na inicialização do PPSSPP
+Stop Sharing = Parar com o compartilhamento
+Stopping.. = Parando...
+
+[Reporting]
+Bad = Ruim
+FeedbackCRCCalculating = CRC do Disco: Calculando...
+FeedbackCRCValue = CRC do Disco: [VALOR]

```

Comment:
`[VALUE]` should be left as it  was as it's used to mark where the actual value should be written in the string

Reply:
Thank you for the tips, it would be good if there was a warning in the language file about that, there is no way that simple translators like me would know about this. ;)